app/src/ep.config.json

```diff
@@ -49,6 +49,7 @@
   },
   "indi": {
     "enable": false,
+    "dashboard": false,
```

Comment:
@rostyk-kanafotskyy This is the wrong section. It should be under `b2b`, not `indi`.